ui/webui/src/components/Error.jsx

```diff
@@ -177,14 +177,18 @@ export const BZReportModal = ({
 
 const addExceptionDataToReportURL = (url, exception) => {
     const newUrl = new URL(url);
+    const backendMessage = exception.backendMessage ? exception.backendMessage + (exception.jsMessage ? " " : "") : "";
```

Comment:
You will probaly need new lines between then backend, jsMessage and stackTrace. Did you try it but I am not sure they are included in the errors (the new lines)

Can you please verify?

Reply:
I checked and there are no newlines at the end of the reports. I added them to the url.